### PR TITLE
Modified Header for Authenticated User

### LINKED ISSUE #18: Update the existing header component.
Refer to the design (Header 2 frame) and add the button and avatar to the existing header components. These two elements will be shown only when the user is logged in. Use states in React.

### PR DESCRIPTION
**Pull request by priyanshisharma**
Fixes #18 
* Passed `isSignedIn` and `avatar` as props.
* Used a conditional block to display, "Write a Post" button and "avatar" image if signed in, else "Sign In" link.
Screen Shot 2020-09-11 at 7 02 57 PM

### CODE REVIEW

File: frontend/src/components/Header.jsx [hunk 1]
```diff
@@ -11,14 +11,29 @@ class Header extends Component {
...
                 <ul className="nav__item-container">
                   <li className="nav__item"><a href="">Stories</a></li>
                   <li className="nav__item"><a href="">Organizations</a></li>
                   <li className="nav__item"><a href="">About</a></li>
-                  <li className="nav__item"><a href="">Login</a></li>
+                    {
+                      this.props.isSignedIn?
+                        <> 
+                          <li className="nav__item">
+                            <button className="button__big" style={{textTransform: "capitalize"}}>
```
Comment (salil-naik) [inline frontend/src/components/Header.jsx:24]: button__small

File: frontend/src/components/Header.jsx [hunk 2]
```diff
@@ -11,14 +11,29 @@ class Header extends Component {
...
+                              Write a Story
+                            </button>
+                          </li>
+                          <li className="nav__item" data-testid="user-avatar">
+                            <img src={this.props.avatarSrc} class="avatar" alt="signed-in user"/>
```
Comment (salil-naik) [inline frontend/src/components/Header.jsx:29]: className

File: frontend/src/css/components/header.scss [hunk 3]
```diff
@@ -41,4 +41,12 @@
       margin-right:20px;
     }
   }
+}
+
+.avatar{
+  width: 40px;
+  height: 40px;
+  left: 436px;
```
Comment (salil-naik) [inline frontend/src/css/components/header.scss:49]: left, top, bottom, right doesn't work without setting the position.

**Review by salil-naik — CHANGES_REQUESTED** [review]

**Comment by priyanshisharma** [comment]
@salil-naik I've just performed the requisite changes, please have a look!

**Review by salil-naik — APPROVED** [review]
Perfect!

**Pull request merged**